grammars/css.cson

```diff
@@ -498,13 +498,11 @@
         'beginCaptures':
           '1':
             'name': 'punctuation.separator.key-value.css'
-        'end': '\\s*(;)|(\\})'
+        'end': '\\s*(;)|\\s*(?=}|\\))'
```

Comment:
No it's not tested. Sure I can, but it'll have to be in another PR for `@supports`, coz the grammar for it is not there yet. Maybe I'll remove the `|\\)` part for now?